### PR TITLE
Update 1907-10-21.xml

### PR DESCRIPTION
**Pull request by emily-seguin**

### CODE REVIEW

**Comment by whanley** [comment]
Thanks Emily. Some tidying is required, because you put some divs inside other divs. Make sure that you close one div before starting the next one. For instance, at line 67 you need a `</div>` to close the nameplate before you start the first ad of the first column. At 326/327 you have two closing divs, when you only need one. (The second closing div actually closes the nameplate). Use the outline panel in Oxygen to see where you're at, and ask for help if you need it.

**Comment by emily-seguin** [comment]
I am currently working on getting OCR to work on my computer. I have finished page 2 of Monday's issue and will update when I get page 3

**Comment by emily-seguin** [comment]
Struggled to post updated version yesterday; still working on page 3 because of OCR issue

**Comment by whanley** [comment]
Hi Emily, this pull request should only include one file. Please delete ` 1907-10-21-pg2-3.xml` and send another pull request.

Generally things look good--you seem to be doing a very fine job of cleaning OCR. Two things:
- most divs should be items or sections, rather than articles. This is not a big deal, but please do use the feature attribute to indicate common features (such as local and general). Full info here: https://dig-eg-gaz.github.io/contents/features/
- line 2291, create a new div to start the article on that page.

**Comment by whanley** [comment]
These files are looking really good. Thanks for your hard work.

A couple of things. In your Monday file, you have many items inside of items--it seems that you are using columns for items? See if you can make sense of what's going on; we can sit down and look this over together.

Don't forget that your ads and templates need to be updated with the content of your particular issues.

File: 1907-10-23.xml [hunk 1]
```diff
@@ -0,0 +1,8070 @@
...
+                    </table>
+                    <p>FOR FURTHER PARTICULARS APPLY TO THE AGENTS OF THE</p>
+                    <p>NORDDEUTSCHER LLOYD at Cairo, Alexandria, Port-Said and Suez.</p>
+                    <p>OTTO STERZING, Agent In Cairo, Opera Square.</p>
+                    <p>C. H. SCHOELLER, Agent In Alexandria, Cleopatra Lane.</p>
+                    <p>Messrs. THOS. COOK &amp; SON (Egypt) LTD., and CARL STANGENS REISEBUREAN are
+                        anthorised to sell tickets in CAIRO and ALEXANDRIA, <measure type="indexNo"
+                            >31-8-905</measure></p>
+                </div>
+                <div type="advert" xm
```
Comment (huoyanzhizhou) [inline 1907-10-23.xml]: 10/23 Line 1769: Don't use div type="feature". Instead, type should be section for Local and General. Also, you need to update some measures of your templates.

**Comment by whanley** [comment]
Lots of stuff here. I'm going to merge these files, and we can tweak them afterwards.

**Comment by whanley** [comment]
Great work. Some polishing required, but thanks for all this!

**Pull request merged**